forge-gui/res/cardsfolder/upcoming/delete.txt

```diff
@@ -0,0 +1,7 @@
+Name:Delete
+ManaCost:X R R
+Types:Sorcery
+A:SP$ DamageAll | Cost$ X R R | ValidCards$ Creature.nonArtifact | ValidPlayers$ Player | NumDmg$ X | SpellDescription$ CARDNAME deals X damage to each nonartifact creature and each player.
```

Comment:
add ValidDescription$ param for better Stack Description

you can remove `Cost param` if you want... unneeded in this case